main/common/aperture.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const createAperture = require('aperture');
+const exitHook = require('async-exit-hook');
```

Comment:
Not sure we want to use this. It's unmaintained and has some issues.

Reply:
Any alternatives? I was just trying to find a package like that that allows async code execution

Reply:
Not that I'm aware.

Reply:
We could ask that `cleanUp` be a sync method, and call that instead when we get an exit signal and then we can use https://github.com/sindresorhus/exit-hook?

Reply:
Or we can just ignore plugins, and we can set the only action when we get a signal be stopping aperture (which is async, so not sure how good that'll go) and then let the whole recover recording logic handle cleanup on next start

I have noticed that when you kill it, Aperture closes the file anyway, and it doesn't really get corrupt. The only time it really messes up is if you force kill or the computer crashes.

Reply:
> and then let the whole recover recording logic handle cleanup on next start

👍🏻

Reply:
So, since Aperture handles closing the recording file, and we're not calling plugins, should I remove this entire exit hook logic? The recover logic at next startup I think covers everything, so I don't think there's need to listen for a kill signal

Reply:
Yes, I think that should be fine.

Reply:
Updated